x509/src/general_name.rs

```diff
@@ -0,0 +1,417 @@
+/// GeneralNames as defined in [RFC 5280 Section 4.2.1.6].
+use crate::alloc::string::ToString;
```

Comment:
Toplevel doc comments should use `//!`

Also the `crate::` on the front of this import is redundant

```suggestion
//! GeneralNames as defined in [RFC 5280 Section 4.2.1.6].

use alloc::string::ToString;
```

Reply:
Fixed.